Sources/Mistica/Components/Lists/Internals/CellLeftSectionView.swift

```diff
@@ -13,6 +13,10 @@ private enum ImageSize {
     static let small: CGFloat = 24
 }
 
+public protocol ListCellContentAssetDelegate: AnyObject {
+    func didTapAsset()
```

Comment:
following the guidelines (like UITableViewDelegate, etc) it should be:
```suggestion
    func listCellContentAssetDidTapOnAsset(_ listCellContentAsset: ListCellContentAsset)
```

Reply:
Done :D

Reply:
you forgot passing itself as a parameter
```diff
- func listCellContentAssetDidTapOnAsset()
+ func listCellContentAssetDidTapOnAsset(_ listCellContentAsset: ListCellContentAsset)
```

Reply:
I didn't added because the class view that have the gesture is an internal wrapper. I don't know if this would add a benefit.